src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -22,6 +22,10 @@ public class SarifCurrentToVersionOneVisitor : SarifRewritingVisitor
         private Run _currentV2Run = null;
         private RunVersionOne _currentRun = null;
 
+        // To understand the purpose of these fields, see the comment on CreateFileKeyIndexMappings.
+        private IDictionary<string, int> _fileKeyToIndexDictionary;
```

Comment:
>_fileKeyToIndexDictionary [](start = 41, length = 25)

can you specify the version-ness of these things in the name? e.g, v1FileKeytoV2Index dictionary? Helps with clarity. For both members please.

Reply:
Fixed.

---
In reply to: [247376669](https://github.com/Microsoft/sarif-sdk/pull/1211#discussion_r247376669) [](ancestors = 247376669)